### ISSUE TITLE
SQL database restores

### ISSUE THREAD

**Issue by HilcorpDBAdmin** [bseltz-cohesity/scripts#67]
Question
Does "restoreSQL.ps1" support logical name (sys.sysfiles.name) as opposed to physical name (sys.sysfiles.filename) for parameter -ndfFolders?

**Comment by bseltz-cohesity** [comment]
No, secondary file path location rules are based on file path/name, not on logical names. See: https://docs.cohesity.com/6_6/Web/UserGuide/Content/MSSQL/SQLRestoreAlternateInstance.htm?tocpath=Databases%7CMS%20SQL%7CRestore%20%2526%20Clone%7CRestore%20MS%20SQL%20Databases%7C_____3#RecoverMSSQLDatabasetoasaNewDatabaseinanAlternateSQLServerInstance:~:text=Click%20Add%20File%20Path%20Rules%20add%20custom%20rules%20for%20the%20data%20files
